packages/db/drizzle/0000_flimsy_ikaris.sql

```diff
@@ -334,6 +334,14 @@ CREATE TABLE IF NOT EXISTS "runbook" (
 	"job_agent_config" jsonb DEFAULT '{}' NOT NULL
 );
 --> statement-breakpoint
+CREATE TABLE IF NOT EXISTS "runbook_job_trigger" (
```

Comment:
not sure why this wasn't here before. I deleted and rebuilt dist and still came up